index.js

```diff
@@ -32,6 +32,7 @@ function validate(file, options, cb) {
           }
         }
       }
+      this.emit('error', new gutil.PluginError(pluginName, 'HTML Validation errors found!'));
       cb();
```

Comment:
How about pass error through callback? Something like:
```
cb('HTML Validation errors found!');
```
Should exit with non-zero error code